modules/rosa-sts-cluster-terraform.adoc

```diff
@@ -0,0 +1,547 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly-terraform.adoc
+//
+ifeval::["{context}" == "rosa-sts-creating-a-cluster-quickly-terraform"]
+:tf-defaults:
+endif::[]
+:_content-type: PROCEDURE
+
+= Creating your ROSA cluster with Terraform
+
+The following Terraform cluster example creates your account-wide IAM roles then creates a ROSA cluster with a managed OIDC configuration.
+
+.Procedure
+. You can grant permissions to your account by using link:https://console.redhat.com/openshift/token[an offline {cluster-manager-first} token]. You set the token as a variable by running the following command:
++
+[source,terminal]
+----
+$ export RHCS_TOKEN=<your_offline_token>
+----
++
+After you have exported your token, you can verify this value by running the following command:
++
+[source,terminal]
+----
+$ echo $RHCS_TOKEN
+----
+
+. Create the Terraform files locally by using the following code templates:
++
+[NOTE]
+====
+These files are created in your current directory. Ensure that you are in the directory where you want to run Terraform.
+====
+
+.. Run the following command to create your `account-roles.tf` file:
++
+[source,terminal]
+----
+$ cat<<-EOF>account-roles.tf
+data "rhcs_policies" "all_policies" {}
+
+data "rhcs_versions" "all" {}
+
+module "create_account_roles" {
+  source  = "terraform-redhat/rosa-sts/aws"
+  version = "0.0.15"
+
+  create_account_roles  = true
+  create_operator_roles = false
+
+  account_role_prefix    = var.account_role_prefix
+  path                   = var.path
+  rosa_openshift_version = regex("^[0-9]+\\\\.[0-9]+", var.rosa_openshift_version)
+  account_role_policies  = data.rhcs_policies.all_policies.account_role_policies
+  all_versions           = data.rhcs_versions.all
+  operator_role_policies = data.rhcs_policies.all_policies.operator_role_policies
+  tags                   = var.additional_tags
+}
+
+resource "time_sleep" "wait_for_role_propagation" {
+  depends_on = [ module.create_account_roles ]
+
+  create_duration = "30s"
+}
+EOF
+----
+
+.. Run the following command to create the `main.tf` file:
```

Comment:
Nitpick, but this line says "the" (followed by command name). I believe the others say "your".